examples/transitions/index.html

```diff
@@ -35,19 +35,15 @@
         getInitialState: function() {
           return {current: 0};
         },
-
```

Comment:
Cool, I also do. I noticed the other examples (see examples/jquery-bootstrap: https://github.com/facebook/react/blob/master/examples/jquery-bootstrap/js/app.js#L19-L31) smush functions together. Would you like me to make these other examples consistent with the whitespaced functions, or just keep the mix?